### PR TITLE
Updates reference documentation with recent additions to kwargs/methods.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by 36000** [comment]
This should happen automatically somehow

File: docs/source/reference/methods.rst [hunk 1]
```diff
@@ -320,7 +372,7 @@ stop:
 
 
 streamlines:
-    full path to the complete, unsegmented tractography file
```
Comment (36000) [inline docs/source/reference/methods.rst:323]: I think this should actually be in kwargs.rst file

Comment (arokem) [reply docs/source/reference/methods.rst:323]: Should have mentioned that these changes happened automatically when I ran `make html` to build the docs.

File: docs/source/reference/methods.rst [hunk 2]
```diff
@@ -320,7 +372,7 @@ stop:
...
+    full path to the complete, unsegmented tractography file Parameters ---------- tractography_ngpus : int, optional Number of GPUs to use in tractography. If non-0, this algorithm is used for tractography, https://github.com/dipy/GPUStreamlines Default: 0
```
Comment (arokem) [inline docs/source/reference/methods.rst:375]: Theres definitely something odd happening here.

**Comment by 36000** [comment]
Superseded by #1002

**Pull request closed without merging**